packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/telecom-telephony-alias-portability-portabilities.routing.js

```diff
@@ -3,17 +3,28 @@ export default /* @ngInject */ ($stateProvider) => {
     url: '/portabilities',
     views: {
       'aliasView@telecom.telephony.alias': {
```

Comment:
`'aliasView@telecom.telephony.alias': 'portabilities'` is the same :)

Reply:
Could you be more explicit? I don't understand what you mean to say.

Reply:
Ah sorry
Instead of writing
```js
'aliasView@telecom.telephony.alias': {
    component: 'portabilities',
}
```
you can write
```js
'aliasView@telecom.telephony.alias': 'portabilities',
```